packs/vsphere/actions/get_vmconsole_urls.yaml

```diff
@@ -1,12 +1,18 @@
 ---
-    name: 'get_vmconsole_urls'
-    runner_type: 'run-python'
-    description: "Retrieves urls of the virtual machines' consoles"
-    enabled: true
-    entry_point: 'get_vmconsole_urls.py'
-    parameters:
-        vms:
-            type: 'array'
-            description: >
-                Coma-separated moId's of the Virtual Machines. e.g. "vm1-,vm-2"
-            required: true
\ No newline at end of file
+  name: 'get_vmconsole_urls'
+  runner_type: 'run-python'
+  description: "Retrieves urls of the virtual machines' consoles"
+  enabled: true
+  entry_point: 'get_vmconsole_urls.py'
+  parameters:
+    vm_ids:
+      type: 'array'
+      description: >
+        Coma-separated moId's of the Virtual Machines. e.g. "vm1-,vm-2"
```

Comment:
I didn't actually write this code.. I just modified it to work with multi endpoints. not sure who the original author is so didn't want to change things just yet.